src/hooks/useEvm/useEvm.ts

```diff
@@ -1,61 +1,70 @@
-import type { ChainId } from '@shapeshiftoss/caip'
-import { fromChainId } from '@shapeshiftoss/caip'
+import { CHAIN_NAMESPACE, type ChainId, isChainReference, toChainId } from '@shapeshiftoss/caip'
 import type { ETHWallet } from '@shapeshiftoss/hdwallet-core'
 import { useCallback, useEffect, useMemo, useState } from 'react'
 import { useWallet } from 'hooks/useWallet/useWallet'
-import { bn } from 'lib/bignumber/bignumber'
 import { getSupportedEvmChainIds } from 'lib/utils/evm'
 import { selectFeatureFlags } from 'state/slices/preferencesSlice/selectors'
 import { useAppSelector } from 'state/store'
 
-const chainIdFromEthNetwork = (
-  ethNetwork: string | undefined,
+const evmChainIdFromChainId = (
```

Comment:
👀

Reply:
The drive for this change is to make the chain menu use our more foxy caip chainId rather than the EVM network ID (caip chain reference). Unfortunately the change leaked into this hook, but overall it seems reasonably nice as it aligns with the vernacular across the rest of the application and is converted at the boundary to metamask. 

Lets discuss this in more detail 🤝